qiita_ware/test/test_ebi.py

```diff
@@ -35,20 +34,16 @@
 
 class TestEBISubmission(TestCase):
     def setUp(self):
-        self.path = join(path.dirname(path.abspath(__file__)), 'test_data',
-                         'test_ebi')
-
-        ebi_test_file = partial(join, self.path)
-
-        self.sample1_fp = ebi_test_file('sample1.fastq.gz')
-        self.sample2_fp = ebi_test_file('sample2.fastq.gz')
-        self.sample3_fp = ebi_test_file('sample3.fastq.gz')
-
         self.temp_dir = gettempdir()
         self.files_to_remove = []
 
+        if not isdir(self.temp_dir):
+            makedirs(self.temp_dir)
+        self.files_to_remove.append(self.temp_dir)
```

Comment:
If gettempdir is set to `/tmp` you will be trying to remove `/tmp` from the system